nixos/lib/testing-python.nix

```diff
@@ -160,7 +157,9 @@ rec {
         # set defaults through environment
         # see: ./test-driver/test-driver.py argparse implementation
         wrapProgram $out/bin/nixos-test-driver \
-          --set startScripts "''${vmStartScripts[*]}" \
+          --set startScripts "${lib.concatStringsSep "," (lib.mapAttrsToList
+            (name: machine: "${name}=${machine.config.system.build.vm}/bin/run-${machine.config.system.name}-vm")
+            nodes)}" \
```

Comment:
Perhaps passing this as JSON is a little cleaner, especially on the python side.

Reply:
Yes, after I was done with the implementation, I also thought JSON is probably cleaner. I'll try to do another pass on the weekend.